resources/rafter/charts/rafter-front-matter-service/README.md

```diff
@@ -0,0 +1,7 @@
+# Rafter Front Matter Service
```

Comment:
I would suggest changing the chart name from `chart/rafter-front-matter-service` to `chart/front-matter-service`. What follows, please update the README title to `# Front Matter Service`.

Reply:
Changed